test/APINotes/basic.swift

```diff
@@ -24,7 +24,7 @@ func testSelectors(a: AnyObject) {
 
 func testSwiftName() {
   moveTo(x: 0, y: 0, z: 0)
-  moveTo(0, 0, 0) // expected-error{{missing argument labels 'x:y:z:' in call}}
+  moveTo(0, 0, 0) // expected-error {{missing argument labels 'x:y:z:' in call}}
```

Comment:
Could you please not modify messages which didn't really change? It makes it a lot easier to see which diagnostics got adjusted and how...

Reply:
I'm sorry. I will review and revise it. I'll be careful from now on.